CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
 **BUG FIXES**
 
 - Fix `enable_efa` parameter validation when using Centos8 and Slurm or ARM instances.
+- Fix AMI/instance types by using the actual cluster AMI for the target architecture      
```

Comment:
I think there is something missing in this sentence. To make it more meaningful for the user shall we mention that we are fixing sanity check for clusters with ARM architectures by using cluster AMI when performing validations?

Reply:
Fixed